osu.Game.Rulesets.Osu/Objects/Drawables/DrawableRepeatPoint.cs

```diff
@@ -19,10 +20,11 @@ public class DrawableRepeatPoint : DrawableOsuHitObject, ITrackSnaking
     {
         private readonly RepeatPoint repeatPoint;
         private readonly DrawableSlider drawableSlider;
+        private readonly Drawable circle;
```

Comment:
This really needs a better name in my opinion. `sliderTail` or just `tail` feel like the natural options.